server/bundler/loader.js

```diff
@@ -134,6 +140,10 @@ function requireTemplate( section ) {
 
 		return acc.concat( [
 			'page( ' + pathRegex + ', function( context, next ) {',
+			'	var envId = ' + JSON.stringify( section.envId ) + ';',
```

Comment:
There is some code duplication here, we can refactor it once we agree that this solution makes sense :)